### PR TITLE
fix: disable auto-pagination for `SearchJobsForAlert` method in talent API

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**
fix: #446

### CODE REVIEW

File: typescript/test/unit/proto.ts [hunk 1]
```diff
@@ -113,10 +113,12 @@ describe('src/schema/proto.ts', () => {
       fd.service[0].method[1] = new plugin.google.protobuf.MethodDescriptorProto();
       fd.service[0].method[1].name = 'SearchProfiles';
       fd.service[0].method[2] = new plugin.google.protobuf.MethodDescriptorProto();
-      fd.service[0].method[2].name = 'ListJobs';
-      fd.service[0].method[2].outputType =
+      fd.service[0].method[2].name = 'SearchJobsForAlert';
```
Comment (alexander-fenster) [inline typescript/test/unit/proto.ts:116]: Can we also add `SearchJobs` here?

File: typescript/src/schema/proto.ts [hunk 2]
```diff
@@ -167,7 +167,9 @@ function pagingField(
   const serviceName =
```
Comment (alexander-fenster) [inline typescript/src/schema/proto.ts:167]: Not directly related to this PR, but while you're here, let's maybe rename these variables to `serviceNameException` and `methodNameException`? It's weird for a variable named like `methodName` to be boolean.

File: typescript/test/unit/proto.ts (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply typescript/test/unit/proto.ts:116]: sure!

File: typescript/src/schema/proto.ts (hunk 2 above)
Comment (xiaozhenliu-gg5) [reply typescript/src/schema/proto.ts:167]: yes, of course. That makes sense!

Comment (xiaozhenliu-gg5) [reply typescript/src/schema/proto.ts:167]: done.

**Pull request merged**